config.md

```diff
@@ -340,6 +340,52 @@ For Linux-based systems, the `process` object supports the following process-spe
+    For more information about Landlock, see [Landlock documentation][landlock].
+    `landlock` contains the following properties:
+
+    * **`ruleset`** (object, OPTIONAL) the `ruleset` field identifies a set of rules (i.e., actions on objects) that need to be handled (i.e., restricted).
+    The `ruleset` currently contains the following types:
+        * **`handledAccessFS`** (array of strings, OPTIONAL) is an array of FS typed actions that are handled by a ruleset.
+        If no rule explicitly allow them, they should then be forbidden.
+        * **`handledAssessNetwork`** (array of strings, OPTIONAL) is an array of NETWORK typed actions that are handled by a ruleset. (The NETWORK typed actions are avaliable when the ABI version >= 4. the behavior of the NETWORK typed actions is not used when the ABI version is less than 4 will depend on the **`disableBestEffort`**)
+    * **`rules`** (object, OPTIONAL) the `rules` field specifies the security policies (i.e., actions allowed on objects) to be added to an existing ruleset.
+    The `rules` currently contains the following types:
+        * **`pathBeneath`** (array of objects, OPTIONAL) is an array of the file-hierarchy typed rules.
+        Entries in the array contain the following properties:
+            * **`allowedAccess`** (array of strings, OPTIONAL) is an array of FS typed actions that are allowed by a rule. The actions are grouped by the ABI version in the following description:
+                1. ABI version >= 1:
+                    1. exectute
+                    2. write_file
+                    3. read_file
+                    4. read_dir
+                    5. remove_dir
+                    6. remove_file
+                    7. make_char
+                    8. make_dir
+                    9. make_reg
+                    10. make_sock
+                    11. make_fifo
+                    12. make_block
+                    13. make_sym
+                2. ABI version >= 2:
+                    1. refer
+                3. ABI version >= 3:
+                    1. truncate
+            * **`paths`** (array of strings, OPTIONAL) is an array of files or parent directories of the file hierarchies to restrict.
+        * **`portBeneath`** (array of objects, OPTIONAL) is an array of the network-hierarchy typed rules.
+        Entries in the array contain the following properties:
+            * **`allowedAccess`** (array of strings, OPTIONAL) is an array of NETWORK typed actions that are allowed by a rule. The actions are grouped by the ABI version in the following description:
+                1. ABI version >= 4:
+                    1. bind
+                    2. connect
+            * **`ports`** (array of strings, OPTIONAL) is an array of network ports to restrict.
+    * **`disableBestEffort`** (bool, OPTIONAL) the `disableBestEffort` field disables the best-effort security approach for Landlock access rights.
```

Comment:
(This is slightly confusing in my mind that the boolean condition has the "inverted" logic here.  I assume that in this config format, it is not possible to call this variable "bestEffortFallback" and to make `true` the default value instead?)